src/css/styles.scss

```diff
@@ -392,7 +392,7 @@ main {
   margin-right: 2%;
 }
 
-#overlay {
+.overlay {
   filter:alpha(opacity=50);
```

Comment:
This made no difference for the bug, but still good to note to keep classes for multiple use rather than id!